core/web-assets/package-lock.json

```diff
@@ -4330,9 +4330,9 @@
             }
         },
         "dot-prop": {
-            "version": "4.2.0",
-            "resolved": "https://registry.npmjs.org/dot-prop/-/dot-prop-4.2.0.tgz",
-            "integrity": "sha512-tUMXrxlExSW6U2EXiiKGSBVdYgtV8qlHL+C10TsW4PURY/ic+eaysnSkwB4kA/mBlCyy/IKDJ+Lc3wbWeaXtuQ==",
+            "version": "4.2.1",
```

Comment:
Guess this is unintentional change

Reply:
> Guess this is unintentional change

I think is must have been. I certainly didn't make it intentionally.

Reply:
That probably came from a merge.  It's made its way through the various branches `foundation-2019` and up.

Reply:
I think you're right @RangerRick . It doesn't show up in a diff done today between the PR branch and `foundation-2019`.